src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -2229,6 +2229,137 @@
             }
           }
         },
+        "enterpriseSearchApplications": {
```

Comment:
@TattdCodeMonkey AN I reading the PR right that this is a plugin migration and _not_ a saved object migration? There isn't a saved object hash change either.
The word "migrate" immediately made me think this was saved object related 😆

Reply:
Correct this is just migrating some pages from one plugin "application" to another one. No saved objects are involved.